ui/src/app/views/dataset-page.tsx

```diff
@@ -332,21 +330,23 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
       this.setState({selectedValues: valuesSelected, dataSetTouched: true});
     }
 
+    get selectAll() {
```

Comment:
if this is not being used anywhere else except selectAllValues we can just directly use it in the if statement

Reply:
Ohh its being used on select so never mind